Utilities/DataCompression/include/DataCompression/TruncatedPrecisionConverter.h

```diff
@@ -18,32 +18,29 @@
 //  @author Matthias Richter
 //  @since  2015-08-08
 //  @brief  A simple converter producing truncated precision
-//          according to a data model 
-
-namespace AliceO2 {
-
-template<class _ParameterModel>
-class TruncatedPrecisionConverter {
-public:
+//          according to a data model
+
+namespace AliceO2
+{
+template <class _ParameterModel>
+class TruncatedPrecisionConverter
+{
+ public:
```

Comment:
The .clang-format configuration is not 100% compliant with our coding guidelines, as some of the rules cannot be replicated. Instead it is more or less 90% accurate and should be used as a pre-formatting tool before manual editing